.swift-format

```diff
@@ -0,0 +1,9 @@
+{
+    "version": 1,
+    "lineLength": 1000,
```

Comment:
Is 1000 intentional here, rather than 100?

Reply:
Yes, we (or rather I) don’t want a maximum line length.

Reply:
Or is there a way to set the maximum line length to inifinity?

Reply:
Is there a way to provide comment line lengths? I'm not sure why, but I do prefer *comments* having a max length. Maybe I'm weird though.

Reply:
We don't currently support that (because who would be crazy enough to want such a thing 😁 ).

But it's a reasonable feature request, since it still ensures that indentation and other rules are consistently applied while honoring the author's own line breaks, but otherwise not forcing any new breaks.

Reply:
> Is there a way to provide comment line lengths? I'm not sure why, but I do prefer _comments_ having a max length. Maybe I'm weird though.

Even when regular line lengths are enforced by swift-format, comments aren't, since we don't want to break weird cases (like cool ASCII art diagrams).

I'd like a more principled way of addressing this, but I'd probably tie it to using swift-markdown to reflow comments and requiring anyone doing the weird edge cases to put them in code blocks that wouldn't get reformatted.

Reply:
Seems reasonable to me, thanks for the info 🙇